[CUBRIDMAN-89] Revise Java SP manual in terms of Java SP server and system environments (#322)
CUBRID/cubrid-manual: en/admin/control.rst

*   status: check status of a database server process. 

| Every command can specify a database name (**[database_name]**) as an argument.

> | If the database name is not specified, the **status** command displays status information of the Java stored procedure server of every currently running database server.

Reviewer: Is it correct to edit here? If it is correct, the korean manual should also be edited.

Author: Thank you. This fix is incorrect. I'm gonna revert it.

Registering Services
--------------------

> You can register database servers, CUBRID brokers, CUBRID Java Stored Procedure servers, CUBRID Manager(s) or CUBRID HA as CUBRID service in the configuration file ( **cubrid.conf** ). To register services, you can input for each **server**, **broker**, **manager** or **heartbeat** as a parameter value, and it is possible to input several values by concatenating them in comma(,).

Reviewer 2: ```suggestion
You can register database servers, CUBRID brokers, CUBRID Java Stored Procedure servers, CUBRID Manager(s) or CUBRID HA as CUBRID service in the configuration file ( **cubrid.conf** ). To register services, you can input each **server**, **broker**, **manager** or **heartbeat** as a parameter value, and it is possible to input several values by concatenating them in comma(,).
```

Author: Thank you.

    -Xrs
    -------------------------------------------------

.. _cubrid-javasp-with-server:

Starting the CUBRID Java SP Server together when the database server starts
------------------------------------------------------------------------------------------

> | If **java_stored_procedure** set to yes for the corresponding database, 

Reviewer 2: ```suggestion
| If **java_stored_procedure** is set to yes for the corresponding database, 
```

Author: Thank you.

| When the database server starts/stops, the Java stored procedure server is started/stopped.

> | The following is an example of Java stored procedure server is started together when the database server starts.

Reviewer 2: ```suggestion
| The following is an example of Java stored procedure server and database server both are started simultaneously.
```

Author: Thank you.

::

    # cubrid.conf

    ...

    [@demodb]
    java_stored_procedure=yes
    
    [@testdb]
    java_stored_procedure=no

    ...

::

    -- demodb's java_stored_procedure is set to yes
    % cubrid server start demodb
    
    @ cubrid server start: demodb

>     This may take a long time depending on the amount of restore works to do.

Reviewer 2: ```suggestion
    This may take a long time depending on the amount of restored work.
```
'restore' is a verb, need to change it. to 'restored'.

Author: This message is just displayed by cubrid utility. We don't need to fix it.

    CUBRID 11.2

    Calling java stored procedure is allowed

::

    -- testdb's java_stored_procedure is set to no
    % cubrid server start testdb
    
    @ cubrid server start: testdb

>     This may take a long time depending on the amount of restore works to do.

Reviewer 2: ```suggestion
    This may take a long time depending on the amount of restored work.
```

Author: This message is just displayed by cubrid utility. We don't need to fix it.

Environment Configuration for Java Stored Function/Procedure
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> To use Java-stored functions/procedures in CUBRID, you must have JDK (Java Development Kit) 1.8 64bit installed in the environment where the CUBRID server is installed. 

Reviewer 3: ```suggestion
To use Java-stored functions/procedures in CUBRID, you must have JDK (Java Development Kit) 1.8 64bit or later installed in the environment where the CUBRID server is installed. 
```

CUBRID first looks for the **jvm.dll** file in the **JVM_PATH** variable. if **JVM_PATH** is not set or if the file cannot be loaded, it looks for the file in the **JAVA_HOME** variable as described above.

*   An example of configuring the **JVM_PATH** environment variable ::
    
    % set JVM_PATH=C:\jdk1.8.0\jre\bin\server\libjvm.dll

**Linux/UNIX Environment**

For Linux/UNIX environment, CUBRID loads the **libjvm.so** file to run the Java Virtual Machine. CUBRID first locates the **libjvm.so** file from the **LD_LIBRARY_PATH** environment variable and then loads it. If it cannot find the file, it uses the **JAVA_HOME** environment variable. For Linux, glibc 2.3.4 or later versions are supported. The following example shows how to configure the Linux environment variable (e.g., **.profile**, **.cshrc**, **.bashrc**, **.bash_profile**, etc.).

*   An example of installing JDK 1.8 and configuring the environment variables in a bash shell ::

    % JAVA_HOME=/usr/java/jdk1.8.0
    % LD_LIBRARY_PATH=$JAVA_HOME/jre/lib/amd64:$JAVA_HOME/jre/lib/amd64/server:$LD_LIBRARY_PATH
    % export JAVA_HOME
    % export LD_LIBRARY_PATH

> *   An example of installing JDK 1.8 and configuring the environment variables in a csh shell ::

Reviewer 4: bash? or csh?

Author: csh is right.
